### PR TITLE
zephyr cmake PLATFORM variable path error

### PR DESCRIPTION
**Pull request by hongshui3000**
fix https://github.com/thesofproject/sof/issues/4047

Signed-off-by: yxh yangxiaohuamail@gmail.com

### CODE REVIEW

**Comment by sofci** [comment]
Can one of the admins verify this patch?
> reply **test this please** to run this test once.

**Comment by gkbldcig** [comment]
Can one of the admins verify this patch?

File: zephyr/CMakeLists.txt [hunk 1]
```diff
@@ -3,6 +3,7 @@
 if(CONFIG_SOF)
```
Comment (dbaluta) [inline zephyr/CMakeLists.txt:3]: Please read this https://chris.beams.io/posts/git-commit/#seven-rules

Your commit message should provide more info to better understand WHY do we need it.!

Comment (marc-hb) [reply zephyr/CMakeLists.txt:3]: Some other very good ones:
https://wiki.openstack.org/wiki/GitCommitMessages
https://wiki.qemu.org/Contribute/SubmitAPatch (Ignore the QEMU-specific advice)
https://wiki.qemu.org/Contribute/SubmitAPullRequest (Ignore the QEMU-specific advice)

**Review by lgirdwood — APPROVED** [review]

**Comment by lgirdwood** [comment]
@zrombel good to merge ? seems to be pending.

**Comment by fredoh9** [comment]
TEST THIS PLEASE

**Pull request merged**